ledger/participant-local-store/src/test/lib/com/daml/platform/localstore/PartyRecordStoreTests.scala

```diff
@@ -72,20 +86,28 @@ trait PartyRecordStoreTests extends PartyRecordStoreSpecBase { self: AsyncFreeSp
       "allow creating a fresh party record" in {
         testIt { tested =>
           for {
-            create1 <- tested.createPartyRecord(newPartyRecord("party1"))
-            create2 <- tested.createPartyRecord(newPartyRecord("party2"))
+            _ <- tested.identityProviderConfigStore.createIdentityProviderConfig(idp1)
```

Comment:
Maybe instead of changing what `tested` is, add a new abstract method `def createIdentityProviderConfig` to the `PartyRecordStoreTests`?

Reply:
I would still need to provide a new container with both new stores, new method should be able to act on this new store then. I dont see how it is possible